namespace-resources/variables.tf

```diff
@@ -0,0 +1,11 @@
+variable "cluster" {
+  description = "What cluster are you deploying your namespace. I.E cloud-platform-test-1 "
```

Comment:
Should we have a default value for live-0 here?

Reply:
Changed